### ISSUE TITLE
Enable discussions in github

### ISSUE THREAD

**Issue by lucas-wilkins** [SasView/sasview#2180]
Can we enable github discussions. Quite often there are topics that require discussion rather than being tickets for things to be done. It would be good to have an appropriate space for them.

Instructions are here: https://docs.github.com/en/discussions/quickstart

**Comment by wpotrzebowski** [comment]
I don't see any reason why we shouldn't do it. I've just enabled them (on organization level)
<img width="1060" alt="Screenshot 2022-09-06 at 09 01 56" src="https://user-images.githubusercontent.com/4189922/188568187-bedad2dc-88ee-4a23-82c0-49e8e331a80c.png">

**Comment by wpotrzebowski** [comment]
@lucas-wilkins can you give a quick intro to github discussion at the next call?

**Comment by butlerpd** [comment]
Wish this had been an option when we started!  Note that currently, at least the two main repos of sasview and sasmodels have a milestone dedicated to such issues (`top level tickets and to discuss` and `To Discuss` respectively). Note that lumping "top level tickets" is because as far as I know github still does not provide child tickets and/or dependency in tickets like most ticketing platforms (though zenhub which I played with briefly does seem to add those concepts to github).

At any rate, my vote would be to remove the discussion milestones and move those issues to the discussion boards?

Also wondering if this could be used to help our long standing issue related to separating the help report questions from the issues stream?  Maybe at the next call?
